odl/solvers/advanced/chambolle_pock.py

```diff
@@ -108,6 +105,39 @@ def chambolle_pock_solver(op, x, tau, sigma, proximal_primal, proximal_dual,
 
     Notes
     -----
+    The problem of interest is:
+
+    .. math::
+
+        \\min_{x \\in X} F(K x) + G(x)
+
+    Where the technical conditions are that :math:`K` is an operator
+    between hilbert spaces :math:`X` and :math:`Y`, where convergence is only
+    guaranteed if :math:`K` is linear and :math:`X, Y` are finite dimensional.
+    Further, :math:`G : X -> [0, +\\infty]` and :math:`F : Y -> [0, +\\infty]`
+    are proper, convex, lower-semicontinuous functionals.
+
+    It is often of interest to study problems that involve several operators,
+    for example the classical TV regularized problem
+
+    .. math::
+
+        \\min_x ||Ax - b||_2^2 + ||\\nabla x||_1
+
+    Here it is often tempting to let :math:`K=A`, :math:`F(y)=||y||_2^2` and
+    :math:`G(x)=||\\nabla x||_1`. This is however not feasible, since the
+    proximal of :math:`||\\nabla .||_1` has no closed form expression.
+
+    Instead, the problem can be formulated :math:`K(x) -> (A(x), \\nabla x)`,
+    :math:`F((x_1, x_2)) -> ||x_1||_2^2 + ||x_2||_1`, :math:`G(x)=0`. See the
+    examples folder for more information on how to do this.
```

Comment:
General comment to this math section: I'm very sceptical to the attempt to keep this part "text editor readable" as far as possible. Some of the constructs used here render really shitty in LaTeX, e.g. `->` looks totally horrible, `||` is acceptable but not great, and the low dot in the norm some lines above also looks awkward.

I'm okay with keeping the math notation somewhat simple, but only if the readability does not suffer. An for this particular section, *online doc readability* comes first IMO, since this is not primarily a developer's section. So at the very least, the arrows and the dot should be proper LaTeX (`\\to` and `\cdot`). The vertical lines are kind of okay, although I don't see a good reason not to use `\\lVert` and `\\rVert`.